astropy/coordinates/orbital_elements.py

```diff
@@ -0,0 +1,243 @@
+     (4, 0,  -1, 1,  833),
+     # second column
+     (0, 0,  1,  -3, 777),
+     (4, 0,  -2, 1,  671),
+     (2, 0,  0,  -3, 607),
+     (2, 0,  2,  -1, 596),
+     (2, -1, 1,  -1, 491),
+     (2, 0,  -2, 1,  -451),
+     (0, 0,  3,  -1, 439),
+     (2, 0,  2,  1,  422),
+     (2, 0,  -3, -1, 421),
+     (2, 1,  -1, 1,  -366),
+     (2, 1,  0,  1,  -351),
+     (4, 0,  0,  1,  331),
+     (2, -1, 1,  1,  315),
+     (2, -2, 0,  -1, 302),
+     (0, 0,  1,  3,  -283),
+     (2, 1,  1,  -1, -229),
+     (1, 1,  0,  -1, 223),
+     (1, 1,  0,  1,  223),
+     (0, 1,  -2, -1, -220),
+     (2, 1,  -1, -1, -220),
+     (1, 0,  1,  1,  -185),
+     (2, -1, -2, -1, 181),
+     (0, 1,  2,  1,  -177),
+     (4, 0,  -2, -1, 176),
+     (4, -1, -1, -1, 166),
+     (1, 0,  1,  -1, -164),
+     (4, 0,  1,  -1, 132),
+     (1, 0,  -1, -1, -119),
+     (4, -1, 0,  -1, 115),
+     (2, -2, 0,  1,  107)
+)
+
+"""Coefficients of polynomials for various terms"""
+_coLc = (2.18316448e+02,  4.81267881e+05, -1.57860000e-03,
+         1.85583502e-06, -1.53388349e-08)
+_coD = (2.97850192e+02,  4.45267111e+05, -1.88190000e-03,
+        1.83194472e-06, -8.84447000e-09)
+_coM = (3.57529109e+02,  3.59990503e+04, -1.53600000e-04,
+        4.08329931e-08)
+_coMc = (1.34963396e+02,  4.77198868e+05,  8.74140000e-03,
+         1.43474081e-05, -6.79717238e-08)
+_coF = (9.32720950e+01,  4.83202018e+05, -3.65390000e-03,
+        -2.83607487e-07,  1.15833246e-09)
+_coA1 = (119.75, 131.849)
+_coA2 = (53.09, 479264.290)
+_coA3 = (313.45, 481266.484)
+_coE = (1.0, -0.002516, -0.0000074)
+
+
+def calc_moon(t):
+    """
+    Lunar position model ELP2000-82 of (Chapront-Touze' and Chapront, 1983, 124, 50)
+
+    This is the simplified version of Jean Meeus, Astronomical Algorithms,
+    second edition, 1998, Willmann-Bell. Meeus claims approximate accuracy of 10"
+    in longitude and 4" in latitude, with no specified time range.
+
+    Tests against JPL ephemerides show accuracy of 10 arcseconds and 50 km over the
+    date range CE 1950-2050.
+
+    Parameters
+    -----------
+    time : `~astropy.time.Time`
+        Time of observation.
+
+    Returns
+    --------
+    skycoord : `~astropy.coordinates.SkyCoord`
+        ICRS Coordinate for the body
+    """
+    # number of centuries since J2000.0.
+    # This should strictly speaking be in Ephemeris Time, but TDB or TT
+    # will introduce error smaller than intrinsic accuracy of algorithm.
+    T = (t.tdb.jyear-2000.0)/100.
+
+    # constants that are needed for all calculations
+    Lc = u.Quantity(polyval(T, _coLc), u.deg)
+    D = u.Quantity(polyval(T, _coD), u.deg)
+    M = u.Quantity(polyval(T, _coM), u.deg)
+    Mc = u.Quantity(polyval(T, _coMc), u.deg)
+    F = u.Quantity(polyval(T, _coF), u.deg)
+
+    A1 = u.Quantity(polyval(T, _coA1), u.deg)
+    A2 = u.Quantity(polyval(T, _coA2), u.deg)
+    A3 = u.Quantity(polyval(T, _coA3), u.deg)
+    E = polyval(T, _coE)
+
+    suml = sumr = 0.0
+    for moon_l_r in _MOON_L_R:
```

Comment:
Looking at this again, I think one can simply write
```
for DNum, MNum, McNum, FNum, LFac, RFac in _MOON_L_R:
```
(and same below)

Sorry that I didn't note that earlier!